pilot/pkg/model/context.go

```diff
@@ -401,20 +401,23 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 	return meshconfig.MeshConfig{
 		MixerCheckServer:                  "",
 		MixerReportServer:                 "",
-		DisablePolicyChecks:               false,
+		DisablePolicyChecks:               true,
 		PolicyCheckFailOpen:               false,
 		SidecarToTelemetrySessionAffinity: false,
 		ProxyListenPort:                   15001,
 		ConnectTimeout:                    types.DurationProto(1 * time.Second),
-		IngressClass:                      "istio",
 		IngressControllerMode:             meshconfig.MeshConfig_STRICT,
```

Comment:
i think this should be disabled instead of enable.. isn't it?

Reply:
Yes the ingressClass is gone now in our helm/istio/templates/configmap.yaml